AFC.py

```diff
@@ -416,9 +416,9 @@ def cmd_PREP(self, gcmd):
                                 while CUR_LANE.load_state == False and num_tries < 20:
                                     CUR_LANE.move( self.hub_move_dis, self.short_moves_speed, self.short_moves_accel)
                                     num_tries += 1
-                                if num_tries > 20:
-                                    message = (' FAILED TO LOAD, CHECK FILAMENT AT TRIGGER\n||==>--||----||-----||\nTRG   LOAD   HUB   TOOL')
-                                    self.handle_lane_failure(CUR_LANE, message)
+                                    if num_tries > 20:
+                                        message = (' FAILED TO LOAD, CHECK FILAMENT AT TRIGGER\n||==>--||----||-----||\nTRG   LOAD   HUB   TOOL')
+                                        self.handle_lane_failure(CUR_LANE, message)
```

Comment:
Hmm...wasn't this correct the way it was? If the while loop only executes when `num_tries < 20`, this condition `if num_tries > 20` would never evaluate to `true`.

Reply:
The if statement was not indented to be in the while loop so loop would sit there forever, but looking at the code again it still wrong as it's missing a break statement to break the loop when num_tries is greater than 20

Reply:
It should naturally stop looping once we hit `num_tries >= 20`. I think this change was targeting the wrong loop. Looking through the prep command, I noticed we have [another loop](https://github.com/ArmoredTurtle/AFC-Klipper-Add-On/blob/63e6d29b82a1a291f2d9ceb3eca02f340a425d1b/AFC.py#L404-L410) that is supposed to break when `x > 20` BUT we never actually increment the `x`. That is likely the infinite loop you ran into.